templates/thank-you.html

```diff
@@ -46,25 +47,36 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
-  {% if content.view_submitted_response.enabled %}
+  {% set countdown_expired_text = _("For security, you can no longer view or get a copy of your answers") %}
+  {% set countdown_text = _("For security, your answers will only be available to view for another ") %}
+
+  {% if content.view_submitted_response.enabled and not content.view_submitted_response.expired %}
     <h1 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }}</h1>
+    <p
+      class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
     {% if not content.view_submitted_response.expired %}
-      <p class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
-      {% call onsPanel({
-        "id": "view-submitted-response-warning",
-        "type": "warn"
-      }) %}
-        <p>{{ _("For security your answers will only be available to view for 45 minutes") }}</p>
-      {% endcall %}
-    {% else %}
-      {% call onsPanel({
-        "id": "view-submitted-response-guidance",
-        "classes": "ons-u-mb-m"
-      }) %}
-        {{ _("For security, you can no longer view your answers") }}
-      {% endcall %}
+      {{ onsTimeoutPanel ({
+          "id": "view-submitted-response-countdown",
+          "minutesTextSingular": _("minute"),
+          "minutesTextPlural": _("minutes"),
+          "secondsTextSingular": _("second"),
+          "secondsTextPlural": _("seconds"),
+          "countdownText": countdown_text,
+          "nojsText": _("For security, your answers will only be available to view for 45 minutes"),
+          "redirectUrl":  url_for("post_submission.get_thank_you"),
+          "countdownExpiredText": countdown_expired_text,
```

Comment:
Just an observation. Slightly odd that this gets shown, then redirect happens. Instead of redirect on zero. This I would think should be configurable if `redirectUrl` wasn't provided. One for DS, but thought I'd highlight in case there was some reasoning behind it.

Reply:
Yes i noticed that too, it shows it briefly. I removed `redirectUrl` , but it still shows up. One to raise with DS